ics-frontend/src/app/header/header.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar color="primary">
+    <mat-toolbar-row>
+        <a mat-button routerLink="/image-classification">Tag your image</a>
+        <span class="spacer"></span>
+        <div class="header-right">
+            <a mat-button routerLink="/image-classification">Analyze</a>
+            <span class="spacer-right"></span>
```

Comment:
You can remove all of these `spacer-right` spans and achieve the same effect by setting a `gap: 40px` inside the `header-right` styles